lua/lvim/core/terminal.lua

```diff
@@ -3,6 +3,8 @@ local Log = require "lvim.core.log"
 
 M.config = function()
   lvim.builtin["terminal"] = {
+    active = true,
```

Comment:
```suggestion
    active = false,
```
We should probably leave this to false

Reply:
Why is the terminal plugin false by default when shortcuts in whichkey require it?
Honest question, I'm fine with false, I just don't recall why this was disabled.

It wasn't false before, but nil, which means that it isn't available in the user's config for autocomplete.

Reply:
It feels weird to add it just for `lazygit`, but I don't really mind either way.